client/src/util/stateManager/world.js

```diff
@@ -242,14 +203,6 @@ export function worldEqUniverse(world, universe) {
   return world.obsAnnotations === universe.obsAnnotations;
 }
 
-export function subsetVarData(world, universe, varData) {
```

Comment:
this was unused?

Reply:
it is obsolete - replaced by Dataframe.subset()

Reply:
got it